shared/haml/user_header.haml

```diff
@@ -36,10 +36,7 @@
 :javascript
   // Use function closure to avoid cluttering global namespace.
   (function() {
-    var cookieKey = '#{'_shortName' + (rack_env?(:production) ? '' : "_#{rack_env}")}';
-    // Because we use rack_env in our cookie key, the client has no way of knowing
-    // what key to look for. Stash it on window so that clients can access this.
-    window.userNameCookieKey = cookieKey;
+    var cookieKey = "#{environment_specific_cookie_name('_shortName')}";
```

Comment:
Ahh, because this shared haml gets used by both pegasus and dashboard, we can't actually use `environment_specific_cookie_name` until/unless we have a helper for pegasus as well. I'm just going to remove this usage for now.